scancode/Dockerfile

```diff
@@ -0,0 +1,35 @@
+# Copyright 2023 The Authors (see AUTHORS file)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM python:3.10
+
+ENV SCANCODE_RELEASE=32.0.8
+
+RUN apt-get update && apt-get install -y bzip2 xz-utils zlib1g libxml2-dev libxslt1-dev
```

Comment:
Just curious, not an opinion. Would it make sense if we build a base image with all these lib installed, and just build from that image. Would it save some time for building images? Is there any specific concerns for why we don't do that?

Reply:
I tested that, the first one is the current way, and the second it use a base image, there is not much difference, plus we will have to maintain WIF and all that. <img width="1488" alt="Screenshot 2023-10-27 at 10 47 19 AM" src="https://github.com/abcxyz/pkg/assets/51539171/50bb1804-de0c-4587-a591-8df4f1832d9e">
<img width="1495" alt="Screenshot 2023-10-27 at 10 46 56 AM" src="https://github.com/abcxyz/pkg/assets/51539171/15cd98f2-74de-44cd-b8d8-e48cc4005103">